lib/validators/strip-styling-on-paste-validator.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Validates stripStylingOnPaste directive option
+ */
+
+import { Validator } from './validator';
+import { ParsedComponent } from '../models';
+
+export class StripStylingOnPasteValidator extends Validator {
+    async validate(): Promise<void> {
+        for (const parsedComponent of Object.values(this.componentSet.components)) {
+            this.validateComponent(parsedComponent);
+        }
+    }
+
+    /**
+     * Validates whether stripStylingOnPaste is set for an editable directive.
+     */
+    validateComponent(parsedComponent: ParsedComponent): void {
+        if (!parsedComponent.directiveOptions) {
+            return;
+        }
+
+        for (const [key, directiveOptions] of Object.entries(parsedComponent.directiveOptions)) {
+            // Only for directives with stripStylingOnPaste option
```

Comment:
You can move the contents of this if block into a new function to reduce nesting and function length.